extension/dynamicconfig/model/schedule_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestScheduleProto(t *testing.T) {
+	metadata := []byte{0x12, 0x23, 0xAF}
+
+	schedule := Schedule{
+		InclusionPatterns: []Pattern{Pattern{}, Pattern{}},
```

Comment:
Let's use real patterns here and check the converted proto.

Reply:
Added patterns